vision/realsense/stream_depth_color.py

```diff
@@ -0,0 +1,81 @@
+"""
+This program will stream color and depth images to a window for the user.
+"""
+
+# First import the library
+import pyrealsense2 as rs
+# Import Numpy for easy array manipulation
+import numpy as np
+# Import OpenCV for easy image rendering
+import cv2
+
+#Initialize resolution and framerate constants
+RESOLUTION_WIDTH = 640
+RESOLUTION_HEIGHT = 480
+FRAMERATE = 30
+
+# Create a pipeline
+pipeline = rs.pipeline()
+
+# Create a config and configure the pipeline to stream
+# different resolutions of color and depth streams (hypothetically)
+config = rs.config()
+#config.enable_device("846112073537") #a unique serial number for a camera, enables config to get data from a specific camera
```

Comment:
Make the serial number a constant and put it up at the top, if it is None do not call the command.